scRNA-seq-advanced/03-dataset_integration.Rmd

```diff
@@ -532,14 +542,13 @@ scater::plotReducedDim(merged_sce,
                        dimred = "harmony_UMAP",
                        # color by broad celltypes
                        colour_by = "celltype_broad",
-                       # Let's also shape by the batches:
-                       shape_by = "sample",
                        # Some styling to help us see the points:
                        point_size = 0.5,
                        point_alpha = 0.2) +
   ggplot2::ggtitle("UMAP after integration with harmony") +
   # Modify the ggplot2 guides so that the legend points are larger and easier to see:
-  ggplot2::guides(colour = ggplot2::guide_legend(override.aes = list(size = 3, alpha = 1)))
+  ggplot2::guides(colour = ggplot2::guide_legend(override.aes = list(size = 3, alpha = 1))) +
+  ggplot2::facet_wrap(~merged_sce$sample, nrow=2)
```

Comment:
In case you missed the comment with an alternative way of doing this on Ally's PR... If you add an `other_fields` argument to `plotReducedDim` you can append `facet_wrap` in a way I find more normal & pleasing. 

```
plotReducedDim(merged_sce,
               ...,
               other_fields = "sample") +
  ggplot2::ggtitle("UMAP after integration with harmony") +
  ggplot2::guides(colour = ggplot2::guide_legend(override.aes = list(size = 3, alpha = 1))) +
  ggplot2::facet_wrap(~ sample, nrow = 3)
```